liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/applications/LooseWarApplication.java

```diff
@@ -1,8 +1,34 @@
+/**
+ * (C) Copyright IBM Corporation 2021.
```

Comment:
This file was introduced in 2019. So can you change this to `2019, 2021`?

Reply:
Done